CUSTOMIZE.md

```diff
@@ -0,0 +1,149 @@
+# Customize
+
+Here we will give you some tips on how to customize the website.
+
+## Table of Contents
```

Comment:
I don't think there's a need for TOC here. let's remove this section?

Reply:
Sure.